homeassistant/components/smhi/strings.json

```diff
@@ -26,12 +26,66 @@
   },
   "entity": {
     "sensor": {
+      "build_up_index": {
+        "name": "Build Up Index"
+      },
+      "drought_code": {
+        "name": "Drought Code"
+      },
+      "duff_moisture_code": {
+        "name": "Duff Moisture Code"
+      },
+      "fine_fuel_moisture_code": {
+        "name": "Fine Fuel Moisture Code"
+      },
+      "fire_weather_index": {
+        "name": "Fire Weather Index"
+      },
+      "forestdry": {
+        "name": "Fuel drying",
+        "state": {
+          "1": "Very wet",
+          "2": "Wet",
+          "3": "Moderate wet",
+          "4": "Dry",
+          "5": "Very dry",
+          "6": "Extemely dry"
+        }
+      },
       "frozen_precipitation": {
         "name": "Frozen precipitation"
       },
+      "fwi": {
+        "name": "Fire Weather Index"
+      },
+      "fwiindex": {
+        "name": "FWI-index",
+        "state": {
+          "1": "Very low risk",
+          "2": "Low risk",
+          "3": "Moderate risk",
+          "4": "High risk",
+          "5": "Very high risk",
+          "6": "Extremely high risk"
```

Comment:
Can we map the states to something that is better templateable? While with these it's a bit easier to understand, with the grassfire one, I don't think anyone would guess that `2` means that the season is over

Reply:
Was fixes but the rebase screwed it up apparently (and I didn't see it).
Now strings fixed (again)